src/auth0.ts

```diff
@@ -26,9 +38,9 @@ class Auth0 {
    * @param  {String} token for Management API
    * @return {Users}
    */
-  users(token) {
+  users(token: string) {
     const {domain, clientId, ...extras} = this.options;
-    return new Users({baseUrl: domain, clientId, ...extras, token});
+    return new Users({baseUrl: domain, ...extras, token});
```

Comment:
Is it expected not to pass `clientId` anymore ?

Reply:
Yes @frederikprijck since this is user API mgmt, we don't need the clientId but we need the token

Reply:
No @frederikprijck we just require token to make these API calls